src/app.go

```diff
@@ -202,7 +202,7 @@ type NvidiaSmiLog struct {
 			EnforcedPowerLimit string `xml:"enforced_power_limit"`
 			MinPowerLimit      string `xml:"min_power_limit"`
 			MaxPowerLimit      string `xml:"max_power_limit"`
-		} `xml:"power_readings"`
+		} `xml:"gpu_power_readings"`
```

Comment:
Wouldn't it be better to do it like that:
```
		} `xml:"power_readings gpu_power_readings"`
```
This would not break compatibility and will match either `power_readings` or `gpu_power_readings`